cupy/random/kernels.py

```diff
@@ -0,0 +1,27 @@
+from cupy import core
+
+_gumbel_kernel = None
+_laplace_kernel = None
+
+
+def _get_gumbel_kernel():
+    global _gumbel_kernel
+    if _gumbel_kernel is None:
+        _gumbel_kernel = core.ElementwiseKernel(
```

Comment:
Please assign in global space.
Creating `ElementwiseKernel` kernel takes only a little time.